src/Common/src/System/Net/SafeCloseSocket.Windows.cs

```diff
@@ -0,0 +1,244 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
For some reason SafeCloseSocket isn't shown as copied. Were there any changes made to this from the original?